src/PowerShellEditorServices/Services/TextDocument/ScriptRegion.cs

```diff
@@ -12,64 +14,42 @@ namespace Microsoft.PowerShell.EditorServices.Services.TextDocument
     /// </summary>
     public sealed class ScriptRegion : IScriptExtent
     {
-        #region Static Methods
+        public TextEdit ToTextEdit() => new() { NewText = Text, Range = ToRange() };
 
-        /// <summary>
-        /// Creates a new instance of the ScriptRegion class from an
-        /// instance of an IScriptExtent implementation.
-        /// </summary>
-        /// <param name="scriptExtent">
-        /// The IScriptExtent to copy into the ScriptRegion.
-        /// </param>
-        /// <returns>
-        /// A new ScriptRegion instance with the same details as the IScriptExtent.
-        /// </returns>
-        public static ScriptRegion Create(IScriptExtent scriptExtent)
```

Comment:
No issues with adding the ctor but we need to keep the public method otherwise it's a binary breaking change.

Reply:
LOL but who on _earth_ would be using it? 👀